docs/current/guides/snippets/use-secrets/security/main.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os
+import anyio
+import dagger
+
+async def main():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+
+        secret_env = client.set_secret("my-secret-var", "secret value here")
+        secret_file = client.set_secret("my-secret-file", "secret file content here")
+
+        # dump secrets to console
+        output = await (
```

Comment:
Small nit 😅: the rest of the guide always uses `out`, while here it's `output`. Same for all languages.

Reply:
done